concent_api/middleman/middleman_server.py

```diff
@@ -0,0 +1,83 @@
+import asyncio
+
+import socket
+import sys
+from logging import getLogger
+
+from middleman.decorators import safe_execution
+
+PORT = 8888
```

Comment:
How about `constants.py` or even django settings ?

Reply:
`constants.py` should be fine... normally, it will not be run without custom port provided ;)

Reply:
I considered making the port mandatory, but decided to have a default after all. We can still change that though if you prefer.

Though I assumed that the default will be 80, but you're right - the default should be just the default port the application uses, not the HTTP port :)